tests/common/eventHandlers.ts

```diff
@@ -0,0 +1,114 @@
+import { ZkVerifyEvents, TransactionStatus } from '../../src';
+
+export interface EventResults {
+    includedInBlockEmitted: boolean;
+    finalizedEmitted: boolean;
+    errorEventEmitted: boolean;
+    attestationConfirmedEmitted?: boolean;
+    attestationBeforeExpectedEmitted?: boolean;
+    attestationMissedEmitted?: boolean;
+    broadcastEmitted?: boolean;
+    unsubscribeEmitted?: boolean;
+}
+
+export const handleCommonEvents = (
+    events: any,
+    expectedProofType: string,
+    expectedDataType: 'verify' | 'vkRegistration',
+): EventResults => {
+    const eventResults: EventResults = {
+        includedInBlockEmitted: false,
+        finalizedEmitted: false,
+        errorEventEmitted: false,
+        attestationConfirmedEmitted: false,
+        attestationBeforeExpectedEmitted: false,
+        attestationMissedEmitted: false,
```

Comment:
What's the difference among these 3 ?

Reply:
If a user signs up to listen to the events and they have specified a specific attestation id to wait for, the the zkverifyjs package will inform the user if an attestation before the one they are waiting for has happened (and continue listening), if the attestation event they received is after their expected value they are waiting for, or whether they event they are waiting for has now occurred.

Reply:
Thanks. Do you think the attestationMissed and attestationBeforeExpected have some use case ?

Reply:
I assume there is a possibility that someone starts listening for events expecting a particular attestation id and they either miss it, or they receive an earlier attestation and have to wait for the next one.

Just informational mostly for the user so they can see what's happening